### ISSUE TITLE
aria-labelledby

### ISSUE THREAD

**Issue by ashlynn4567** [AlaskaAirlines/Icons#189]
## Describe the bug

When using auro-hyperlink in our project, we were flagged by our accessibility checker that this component was not in compliance. The issue is that the property `aria-labelledby` is misspelled as `aria-labeledby`, and therefore it's accessibility properties cannot be used. 

## Verify the version of @aurodesignsystem/auro-hyperlink you have installed

**The current version is:**
[![See it on NPM!](https://img.shields.io/npm/v/@aurodesignsystem/auro-hyperlink?style=for-the-badge&color=orange)](https://www.npmjs.com/package/@aurodesignsystem/auro-hyperlink)

My project has @aurodesignsystem/auro-hyperlink v3.3.2 installed

## To Reproduce

If you wish to reproduce seeing this issue in our accessibility checker, we use [axe devTools](https://chrome.google.com/webstore/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd?hl=en-US). It is available as an extension to your browser. 

1. Download the axe devTools extension.
1. Go to your page that uses auro-hyperlink as a component. 
1. Right click on the page and select "inspect" to open your browser's devTools.
1. Open the axe devTools tab.
1. Select the option to scan all of your page.
1. Scroll through the accessibility flags to find the one that mentions "ARIA attributes must conform to valid names".

Are you able to reproduce this issue on the [Auro doc site](https://auro.alaskaair.com/)?

- [x] YES
- [ ] NO

## Screenshots
<img width="617" alt="image" src="https://user-images.githubusercontent.com/97701682/223189689-1d4d50bd-cc89-487c-a9e7-e0649f45bf25.png">
<img width="615" alt="image" src="https://user-images.githubusercontent.com/97701682/223190811-a6affaa7-b854-46e3-a69a-ccf02d0d074d.png">
<img width="1322" alt="image" src="https://user-images.githubusercontent.com/97701682/223188859-a50c1a1c-c25a-4fe0-b6d7-590bae5a361a.png">
<img width="1304" alt="image" src="https://user-images.githubusercontent.com/97701682/223188768-b5d56e03-553f-4490-bca2-b8fb55b9a161.png">

## Desktop:

 - OS: macOS v13.2.1 
 - Browser: Chrome v110.0.5481.100

## Additional context
N/A
